core/src/main/java/org/apache/calcite/plan/RelOptAbstractTable.java

```diff
@@ -97,6 +97,11 @@ public boolean isKey(ImmutableBitSet columns) {
     return false;
   }
 
+  // Override to get unique keys
+  public List<ImmutableBitSet> getKeys() {
+    return null;
```

Comment:
Should we consider giving it empty list instead of null?

Reply:
See getReferentialConstraints and getCollationList

Reply:
Thanks for reviewing  @hsyuan 
PR updated.